src/Behat/Cores/Drupal8.php

```diff
@@ -469,4 +469,23 @@ public function createFileUrl($file, bool $relative = TRUE) {
     throw new InvalidArgumentException('%s method only accept %s objects in Drupal 8 or higher', __METHOD__, FileInterface::class);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getLanguagePrefix($language) {
+    $language_manager = \Drupal::languageManager();
+    $language_list = $language_manager->getStandardLanguageList();
+    array_walk($language_list, function (&$item1, $key) {
+      $item1 = current($item1);
+    });
+    $language_list = array_flip($language_list);
+    $langcode = $language_list[$language] ?? '';
+    if ($langcode) {
```

Comment:
I think it's clear if the "if" checks for an error and then throws an exception. Ending the function throwing an exception looks weird to me.

```
  if ($langcode === '') {
    throw new \InvalidArgumentException(sprintf("Language %s not found", $language));  
  }

  $prefixes = \Drupal::config('language.negotiation')->get('url.prefixes');
  return $prefixes[$langcode] . '/';

```
The idea is an error condition is checked, and if true throw exception, continue if not.